test/replication/gh-3055-election-promote.result

```diff
@@ -10,7 +10,7 @@ SERVERS = {'election_replica1', 'election_replica2', 'election_replica3'}
  | ---
  | ...
 -- Start in candidate state in order for bootstrap to work.
-test_run:create_cluster(SERVERS, 'replication', {args='2 0.1 candidate'})
+test_run:create_cluster(SERVERS, 'replication', {args='2 0.1 candidate 3 off'})
```

Comment:
You may omit the default values here and in other places below

Reply:
At first I thought so too. But there seems to be a [bug in test-run](https://github.com/tarantool/test-run/issues/347), which affects this particular line and one in replication/gh-5445-leader-inconsistency.test.lua. To be on the 'safe' side, I've just implicitly send all the arguments.

<details>
<summary> Another currently possible patch</summary>

```diff
diff --git a/test/replication/election_replica.lua b/test/replication/election_replica.lua
index 4848775f2..2ae29877a 100644
--- a/test/replication/election_replica.lua
+++ b/test/replication/election_replica.lua
@@ -6,7 +6,7 @@ local SYNCHRO_QUORUM = arg[1] and tonumber(arg[1]) or 3
 local ELECTION_TIMEOUT = arg[2] and tonumber(arg[2]) or 0.1
 local ELECTION_MODE = arg[3] or 'candidate'
 local CONNECT_QUORUM = arg[4] and tonumber(arg[4]) or 3
-local ELECTION_FENCING_ENABLED = arg[5] and true or false
+local ELECTION_FENCING_MODE = arg[5] or 'off'

 local function instance_uri(instance_id)
     return SOCKET_DIR..'/election_replica'..instance_id..'.sock';
@@ -25,7 +25,7 @@ box.cfg({
     replication_connect_quorum = CONNECT_QUORUM,
     election_mode = ELECTION_MODE,
     election_timeout = ELECTION_TIMEOUT,
-    election_fencing_enabled = ELECTION_FENCING_ENABLED,
+    election_fencing_mode = ELECTION_FENCING_MODE,
     replication_synchro_quorum = SYNCHRO_QUORUM,
     replication_synchro_timeout = 0.1,
     -- To reveal more election logs.
diff --git a/test/replication/gh-3055-election-promote.result b/test/replication/gh-3055-election-promote.result
index 50df58973..a0a69ae3e 100644
--- a/test/replication/gh-3055-election-promote.result
+++ b/test/replication/gh-3055-election-promote.result
@@ -10,7 +10,7 @@ SERVERS = {'election_replica1', 'election_replica2', 'election_replica3'}
  | ---
  | ...
 -- Start in candidate state in order for bootstrap to work.
-test_run:create_cluster(SERVERS, 'replication', {args='2 0.1 candidate'})
+test_run:create_cluster(SERVERS, 'replication', {args='2 0.1'})
  | ---
  | ...
 test_run:wait_fullmesh(SERVERS)
diff --git a/test/replication/gh-3055-election-promote.test.lua b/test/replication/gh-3055-election-promote.test.lua
index 45e48f21b..13324c829 100644
--- a/test/replication/gh-3055-election-promote.test.lua
+++ b/test/replication/gh-3055-election-promote.test.lua
@@ -5,7 +5,7 @@ test_run = require('test_run').new()
 -- in order to promote it to leader.
 SERVERS = {'election_replica1', 'election_replica2', 'election_replica3'}
 -- Start in candidate state in order for bootstrap to work.
-test_run:create_cluster(SERVERS, 'replication', {args='2 0.1 candidate'})
+test_run:create_cluster(SERVERS, 'replication', {args='2 0.1'})
```
</details>

Reply:
Ok, I see now. I'm fine with passing all the arguments explicitly then